src/odemis/driver/omicronxx.py

```diff
@@ -420,8 +420,9 @@ def GetSpecInfo(self):
         ans = self._getValue("GSI")
         # Expects something like:
         # GSI [m63] (optional) int (wl in nm) § int (power in mW)
```

Comment:
I tested both patterns with this example string, and no matches were found. Am I misinterpreting this string? I tried: `"GSI [m63] (optional) int (wl in nm) § int (power in mW)"`, `"GSI [m63] int 900 § int 900"` and `"GSI [m63] int (900) § int (900)"`

Reply:
No, it wasn't meant to be taken so literally. There is no space and "int" has to be replaced by a number. Like:
`GSI[m63]550§1564`
I can add this example, if you think that helps :-)